game_eggs/steamcmd_servers/ark_survival_ascended/README.md

```diff
@@ -0,0 +1,28 @@
+# ARK: Survival Ascended
+
+Steam Description :
+ARK is reimagined from the ground-up into the next-generation of video game technology with Unreal Engine 5! Form a tribe, tame & breed hundreds of unique dinosaurs and primeval creatures, explore, craft, build, and fight your way to the top of the food-chain. Your new world awaits!
+
+## Troubleshooting
+
+Crash logs are available in the Files `ShooterGame/Saved/Crashes/UECC-Windows-*/CrashContext.runtime-xml`
+
+
+## Recommended server settings
+
+### Minimum RAM
+
+This server requires about 8-10GB of RAM to run with no players on a default map.
+
+See the following - <https://ark.wiki.gg/wiki/Dedicated_server_setup#RAM>
+
+## Server Ports
+
+Note: RCON doesn't need to be public but is required for console commands (in ptero)
+
+| Port            | Default |
+| --------------- | ------- |
+| Game            | 7777    |
+| Raw UDP         | 7778    |
+| Query           | 27015   |
+| RCON            | 27020   |
```

Comment:
`37015` is the current egg default. I personally use 27020 but not sure if this should get updated to reflect what the egg does by default?

Reply:
Or add a new only for local ? The other is like if u want to access Rcon from outside right ?

Reply:
> Or add a new only for local ? The other is like if u want to access Rcon from outside right ?

You can only have 1 RCON port. It's just optional to assign the RCON port to a public IP vs `127.0.0.1` for internal/console use.